src/views/Login/index.tsx

```diff
@@ -15,8 +18,9 @@ const Login = () => {
         <OktaSignInWidget
           onSuccess={response => {
             authService.getTokenManager().then((tokenManager: any) => {
-              tokenManager.add('idToken', response);
-              tokenManager.add('accessToken', response);
+              tokenManager.add('idToken', response.token.idToken);
```

Comment:
I don't think this `onSuccess` is actually executing. I added a `console.log` and I don't see anything happening here.

https://github.com/okta/okta-signin-widget/issues/1262

Reply:
Ok. Do you think we should remove this code for now?

Reply:
Possibly. My comment was intended to be informational. Since this code block is never getting executed, it might be doing something different from what you were intending.

I don't see a reason to keep it if it's not doing anything